Sources/HAP/Base/Accessory.swift

```diff
@@ -30,6 +35,34 @@ open class Accessory {
     public let type: AccessoryType
     public let info: Service.Info
     internal let services: [Service]
+    
+    // An accessory implementation can set this flag to false if a device becomes unreachable for a period
```

Comment:
Please wrap comments (throughout) at 80 characters and end sentences with a period.